Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ruby:2.4-alpine
+
+RUN apk --update add grep
+
+WORKDIR /usr/src/app
+COPY . /usr/src/app
+
+RUN gem install -g
```

Comment:
Is this an alternative way of invoking `bundle install`? If so, what is the benefit of using this over `bundle install`?